static/css/pad.css

```diff
@@ -150,7 +150,12 @@ li[data-key=showusers] > a #online_count {
 
 .toolbar ul li a .buttonicon:focus {
     border: 0;
-    outline: none;
+}
+
```

Comment:
I think it would also be good if the focus style of the toolbar buttons was the same as the hover style